frontend/src/app/components/ExperimentPage.js

```diff
@@ -18,7 +18,25 @@ import ExperimentPreFeedbackDialog from '../components/ExperimentPreFeedbackDial
 
 const CHANGE_HEADER_ON = 105;
 
+
 export default class ExperimentPage extends React.Component {
+  render() {
+    const { getExperimentBySlug, params } = this.props;
+    const experiment = getExperimentBySlug(params.slug);
```

Comment:
Hm, not sure I agree with that. Information from the router just more state from the store, right? Why is it bad for our components to depend on that? Especially if it's [easily testable](https://github.com/mozilla/testpilot/pull/1587/files#diff-fdc1fc59bc074c06a5e102348cc58cf5R12).